scripts/macpython-install-python.sh

```diff
@@ -405,16 +408,16 @@ function make_workon_venv {
 if test "$(arch)" == "arm64"; then
   echo "we are arm"
   PLAT=arm64
-  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
+  for pyversion in $LATEST_3p9 $LATEST_3p10 $LATEST_3p11 $LATEST_3p12; do
     install_macpython $pyversion 11
     install_virtualenv
   done
 else
-  for pyversion in $LATEST_3p7; do
-    install_macpython $pyversion 10.9
-    install_virtualenv
-  done
-  for pyversion in $LATEST_3p8 $LATEST_3p9 $LATEST_3p10 $LATEST_3p11; do
+  #for pyversion in $LATEST_3p7; do
+    #install_macpython $pyversion 10.9
+    #install_virtualenv
+  #done
```

Comment:
Remove?

Reply:
I thought it would be helpful to leave as a reminder that 3.7 is associated with 10.9.

Reply:
Ah, I didn't catch that. Would you please add a one-line comment to that effect?